pkg/services/pluginsintegration/pluginconfig/request.go

```diff
@@ -17,8 +18,13 @@ import (
 
 var _ PluginRequestConfigProvider = (*RequestConfigProvider)(nil)
 
+type PluginInfo struct {
+	ID              string
+	ExternalService *auth.ExternalService
```

Comment:
Is it only ever `ClientSecret` that we need or is worth having the whole struct?

Reply:
right now is only the ID and `ClientSecret` but in the future it could be more :shrug: We can omit the struct.

Reply:
Whatever you think makes the most sense - just wanted to check in case you knew for certainty 👍